package.json

```diff
@@ -20,6 +21,10 @@
     "react-icons": "^5.0.1"
   },
   "devDependencies": {
+    "@commitlint/cli": "^19.4.1",
+    "@commitlint/config-conventional": "^19.4.1",
+    "@commitlint/cz-commitlint": "^19.4.0",
+    "@commitlint/prompt-cli": "^19.4.1",
```

Comment:
Those are not needed, do you mind to remove them? Please remind to update also the `package-lock.json`.